reference-architecture/aws-ansible/playbooks/roles/rhsm-repos/tasks/main.yaml

```diff
@@ -1,11 +1,26 @@
 ---
 - name: disable unneeded repos
   command: subscription-manager repos --disable='*'
-  when: register_result | changed
+  when: register_result | changed and ansible_distribution == "RedHat"
```

Comment:
Where is register_result coming from? relying on a registered variable from another role seems a bit fragile.